browser/src/Services/Configuration/IConfigurationValues.ts

```diff
@@ -150,6 +150,9 @@ export interface IConfigurationValues {
     // a custom init.vim, as that may cause problematic behavior
     "editor.completions.mode": string
 
+    // Decide whether or not the completion matching should be case sensitive
+    "editor.completions.caseSensitive": boolean
```

Comment:
Thanks for the PR @bschulte!

It may make sense to make this like the other case sensitive option, ie `string | boolean` such that it can be `smart`/`true`/`false`.

Then the check becomes something like this:

https://github.com/onivim/oni/blob/262098fefd7e82bb69b0b9b68ff66ba6af3d18d8/browser/src/Services/Menu/MenuFilter.ts#L25-L39

That should also let you simplify the regex check like we have over here:

https://github.com/onivim/oni/blob/afab2beac8c9374db9996118d1892e2bf074275d/browser/src/Services/QuickOpen/RegExFilter.ts#L34-L38

That lets us use the same regex for all three cases then.